src/js/view/reducers/logViewerReducer.js

```diff
@@ -183,16 +185,54 @@ export const logViewerReducer = (state = initialState, action) => {
 
     case 'LOGVIEWER_ADD_FILTERED_LINES': {
       console.log('ADDING FILTERED LINES');
-      const { sourcePath, filteredLines, lineCount } = action.data;
+      const { sourcePath, filteredLines, filterString } = action.data;
+      const previousLines = state.filteredLogs[sourcePath]
+        ? state.filteredLogs[sourcePath]
+        : [];
+      const previousFilterString = state.filterString[sourcePath]
+        ? state.filterString[sourcePath]
+        : '';
+
+      console.log({ previousLines, previousFilterString });
```

Comment:
Nice, just tried it out and so far so good. Found a stray console.log here that could perhaps be removed. Will try it some more and look at the code.